src/components/Item/Item.js

```diff
@@ -9,20 +11,65 @@ import TotalPrice from './TotalPrice';
 import DiscountPrice from './DiscountPrice';
 import PriceBox from './PriceBox';
 import Wrapper from './Wrapper';
+import Details from './Details';
 
-function Item() {
+const getTotalPrice = function(item) {
+  const { quantity, price } = item;
+  return quantity * (price.regular - price.discount);
+};
+
+const renderAttributes = function(item) {
+  const { variants } = item;
+
+  if (!variants) {
+    return null;
+  }
+
+  const size = variants.size ? variants.size : NO_SIZE;
+  const color = variants.color ? variants.color : NO_COLOR;
+
+  return (
+    <>
+      <Attribute>Size: {size}</Attribute>
+      <Attribute>Color: {color}</Attribute>
+    </>
+  );
+};
+
+const renderPrices = function(item) {
+  if (!item.price) {
+    return null;
+  }
+  const { price } = item;
+
+  return (
+    <PriceBox>
+      <DiscountPrice>Sale ${price.discount}</DiscountPrice>
+      <Price>Regular ${price.regular}</Price>
+      <TotalPrice>Total ${getTotalPrice(item)}</TotalPrice>
+    </PriceBox>
+  );
+};
+
+function Item({ item = {}, editableQuantity = false, handleIncrease, handleDecrease, handleUpdate, handleRemove }) {
   return (
     <Wrapper>
-      <Image src="https://images-na.ssl-images-amazon.com/images/I/71ZgPluVoTL._UX385_.jpg" />
-      <Title>Men's Chaps Performance Series Classic-Fit 4-Way Strech Sit Jacket</Title>
-      <Attribute>Size: 42 SHORT</Attribute>
-      <Attribute>Color: Charcoal</Attribute>
-      <PriceBox>
-        <DiscountPrice>Sale $125.00</DiscountPrice>
-        <Price>Regular $240.00</Price>
-        <TotalPrice>Total $115.00</TotalPrice>
-      </PriceBox>
-      <Quantity />
+      <Image src={item.imgUrl} />
+      <Details>
+        <Title>{item.name}</Title>
+        {renderAttributes(item)}
+        {renderPrices(item)}
+      </Details>
+      {editableQuantity ? (
```

Comment:
Don't perform conditionals in `JSX`. Create spearate `renderQuantity` method, that will handle the logic and render.